presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -55,6 +55,17 @@ export namespace SelectClassInfo {
       relatedInstanceClasses: json.relatedInstanceClasses.map((p) => RelatedClassInfo.fromJSON(p)),
     };
   }
+
+  export function fromCompressedSelectClassInfoJSON(compressedSelectClass: SelectClassInfoJSON<string>, classesMap: { [id: string]: CompressedClassInfoJSON }): SelectClassInfoJSON {
```

Comment:
The function is already under `SelectClassInfo` namespace - no need to repeat that in function name